index.js

```diff
@@ -1,7 +1,20 @@
 const mjml = require('mjml');
+const colors = require('colors');
```

Comment:
I'm using this package to colourize the outputted error. The documentation for the [color](https://www.npmjs.com/package/colors) package notes that it extends `String.prototype` hence why I could do `.red` on a string on [Line 10](https://github.com/ramasilveyra/mjml-loader/pull/1/files/980c1381b4ae14b5d28159921be3ebf0da115aaf#diff-168726dbe96b3ce427e7fedce31bb0bcR10) 😆. But I'll make it more explicit by using the method.